kolibri/plugins/user_profile/assets/src/views/ChangeFacility/UsernameExists.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+
+  <div>
+    <h1>{{ $tr('documentTitle') }}</h1>
+    <p>{{ firstLine }}</p>
+    <p>{{ secondLine }}</p>
+    <BottomAppBar>
+      <slot name="buttons">
+        <KButtonGroup>
+          <KButton
+            :primary="false"
+            :text="$tr('createAccount')"
+            appearance="flat-button"
+            data-test="createButton"
+            @click="to_create"
+          />
+          <KButton
+            :primary="true"
+            :text="$tr('mergeAccounts')"
+            data-test="mergeButton"
+            @click="to_merge"
+          />
+        </KButtonGroup>
+      </slot>
+    </BottomAppBar>
+
+  </div>
+
+</template>
+
+
+<script>
+
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import BottomAppBar from 'kolibri.coreVue.components.BottomAppBar';
+  import { computed } from 'kolibri.lib.vueCompositionApi';
+  import get from 'lodash/get';
+
+  export default {
+    name: 'UsernameExists',
+    metaInfo() {
+      return {
+        title: this.$tr('documentTitle'),
+      };
+    },
+    components: { BottomAppBar },
+
+    mixins: [commonCoreStrings],
+    setup() {
+      const firstLine = computed({
+        get() {
+          return this.$tr('changeFacilityInfoLine1', {
+            target_facility: get(this.state, 'value.targetFacility.name', ''),
+            username: get(this.state, 'value.username', ''),
+          });
+        },
+      });
+
+      const secondLine = computed({
+        get() {
+          return this.$tr('changeFacilityInfoLine2', {
+            target_facility: get(this.state, 'value.targetFacility.name', ''),
+          });
+        },
+      });
+
+      function to_create() {
+        this.changeFacilityService.send({
+          type: 'NEW',
+        });
+      }
+
+      function to_merge() {
+        this.changeFacilityService.send({
+          type: 'MERGE',
+        });
+      }
+
+      return { firstLine, secondLine, to_create, to_merge };
+    },
+
+    inject: ['changeFacilityService', 'state'],
```

Comment:
Can set this up using the composition API too - can import `inject` from `'kolibri.lib.vueCompositionApi'`, and then assign it to a variable inside `setup`:
```
const changeFacilityService = inject('changeFacilityService');
```

No need to reference this inside the methods you are creating in that case.

Reply:
Good advice, thanks. Done!